config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -127,6 +127,8 @@ presubmits:
         - --build=bazel
         - --cluster=
         - --env=KUBE_FEATURE_GATES=AllAlpha=true,CSIMigration=false
+        - --env=KUBE_PROXY_DAEMONSET=true
```

Comment:
Seems like this should probably be pushed back to the 1.16 branch as well.

Reply:
Do you mean this particular line or whole PR?